apps/src/music/appDefaults.js

```diff
@@ -0,0 +1,7 @@
+import {queryParams} from '@cdo/apps/code-studio/utils';
```

Comment:
Maybe something along the lines of "appConfig" to highlight its role as providing configuration values?

Reply:
Good call; done.